### ISSUE TITLE
Not working on New York Times

### ISSUE THREAD

**Issue by AndyTheFactory** [AndyTheFactory/newspaper4k#394]
<a href="https://github.com/JohnChu101"><img src="https://avatars.githubusercontent.com/u/3501811?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [JohnChu101](https://github.com/JohnChu101)**
_Thu Aug 22 03:34:16 2019_
_Originally opened as https://github.com/codelucas/newspaper/issues/729_

----

As mentioned in many issues: #645 #363 , newspaper doesn't work on New York times.
And I tested two versions of New York times, one is the English version, the second is the Chinese version (https://cn.nytimes.com).
The Chinese version doesn't have payment wall, so newspaper should be able to extract the full content of it. However in both cases, newspaper only extract like 3 or 4 paragraphs and they are not from the beginning.
Is there any way i can solve this?
Thanks.

My code:
```
from newspaper import Article, Config as NewspaperConfig
url="https://www.nytimes.com/2019/08/21/business/economy/jobs-growth-revision.html"
conf = NewspaperConfig()
article = Article(url, config=conf, keep_article_html=True)
article.download()
article.parse()
print(article.article_html)
print(article.text)
```

The urls i tested with:
https://www.nytimes.com/2019/08/21/business/economy/jobs-growth-revision.html
https://cn.nytimes.com/china/20190821/china-hong-kong-social-media-soft-power/
https://cn.nytimes.com/morning-brief/20190822/hong-kong-protests-british-consulate-us-sanctions-fentanyl/

**Comment by AndyTheFactory** [comment]
The Chinese article does not work because is_highlink_density does not use the proper tokenizer to split non-european languages. 
Therefore many divs appear as high link density

Must re-think the multi-language handling

**Comment by AndyTheFactory** [comment]
fixed in 0.9.3
